RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -2190,6 +2195,7 @@
 				2D1015DA275959840086173F /* StoreTransaction.swift in Sources */,
 				B34605BC279A6E380031CA74 /* CallbackCache.swift in Sources */,
 				B3E26A4A26BE0A8E003ACCF3 /* Error+Extensions.swift in Sources */,
+				A52004CB2817553200AAFE65 /* PurchasesDeprecation.swift in Sources */,
```

Comment:
are we okay with doing this? i could make a different PurchasesDeprecation for the Purchases module and only expose this one value there... or does anyone have other ideas for avoiding the build failure/warning on calling something deprecated?

Reply:
I'm not sure what you mean. Which build failure?

Reply:
the CI build was failing because of the xcode build warning on calling the deprecated `automaticAppleSearchAdsAttributionCollection` within Purchases.swift: https://app.circleci.com/pipelines/github/RevenueCat/purchases-ios/6418/workflows/1c8d28f1-5600-428c-b11d-0e81db1c8f94/jobs/24363

we made a protocol to avoid deprecation warnings from within our unit tests

Reply:
though the fix appears to not be working anyways. i'll un-draft this PR when it's ready...

Reply:
Deprecations are warnings, obsoletions are build errors. I guess we have something that ensures the build doesn't compile with warnings?
Do we need to call the deprecated method?

Reply:
yeah, it looks like we fail on build warnings.

and yes -- it needs to be called on de-init...